scaladoc/resources/dotty_res/styles/scalastyle.css

```diff
@@ -199,19 +199,21 @@ th {
   display: flex;
   align-items: center;
   flex: 1;
-  overflow-x: hidden;
   overflow-wrap: anywhere;
   color: var(--leftbar-fg);
   margin-right: .5rem;
-  padding-top: 3px;
-  padding-bottom: 3px;
-  margin-top: 1px;
-  margin-bottom: 1px;
-  width: 100%;
-  /* This trick adds selected background stretching to the left side of screen */
-  margin-left: calc(0px - var(--side-width));
-  padding-left: var(--side-width);
-  width: calc(2 * var(--side-width));
+  padding-top: 2%;
+  padding-bottom: 2%;
+  width: calc(var(--side-width));
+  padding-left: calc(0.05 * var(--side-width));
+  padding-right: calc(0.3 * var(--side-width));
+  box-sizing: border-box;
+}
+
+#sideMenu2 a span:not(.micon) {
+  padding-left: 1.5em;
+  text-indent: -1.5em;
+  width: calc(0.5 * var(--side-width));
```

Comment:
The indentation was there intentionally to make wrapped entries look clearer.

Reply:
> > Word-wrapping seem to work fine in Firefox and Chrome without all the hints generated by `Ux`, I think it can be safely removed.
> 
> But it doesn't work properly for package names and long class names because there's no spaces and browser wraps text in random place. Wrapping before dots and upper cases looks better for me

Oh so that was the point! In that case yes, I agree it's useful.

Reply:
Hmm OK why not